lib/static.js

```diff
@@ -95,7 +95,10 @@ module.exports = async () => {
 
   await writeMarkupFiles(getPath(), staticDir);
 
-  const assets = _.compact(_.flatten([options.scripts, options.css]));
+  const assets = _.compact(_.flatten([
+    typeof options.scripts === 'string' ? options.scripts.split(',') : options.scripts,
+    options.css
```

Comment:
Thanks for your fix, @greg0ire! Could you please apply the same fix to `options.css`?

Reply:
Done. I also fixed some eslint issues I had introduced, it seems.